echopype/convert/set_groups_azfp.py

```diff
@@ -27,14 +27,14 @@ def set_env(self) -> xr.Dataset:
         """Set the Environment group.
         """
         # TODO Look at why this cannot be encoded without the modifications -- @ngkavin: what modification?
-        ping_time = (self.parser_obj.ping_time - np.datetime64('1970-01-01T00:00:00')) / np.timedelta64(1, 's')
+        ping_time = (self.parser_obj.ping_time - np.datetime64('1900-01-01T00:00:00')) / np.timedelta64(1, 's')
         ds = xr.Dataset({'temperature': (['ping_time'], self.parser_obj.unpacked_data['temperature'])},
                         coords={'ping_time': (['ping_time'], ping_time,
                                 {'axis': 'T',
                                  'calendar': 'gregorian',
                                  'long_name': 'Timestamp of each ping',
                                  'standard_name': 'time',
-                                 'units': 'seconds since 1970-01-01'})},
+                                 'units': 'seconds since 1900-01-01'})},
```

Comment:
I suggest changing the datetime string to a fully explicit one: 1900-01-01T00:00:00Z

Reply:
Shouldn't it be `seconds since 1900-01-01 00:00:00` rather than isoformat?

Reply:
I'm proud of you @lsetiawan , for going to the source :smile:! The iso8601 spec is a bit squishy in that allows for some variations; for example, you can use a blank space instead of the T separator, and either a UTC offset or a time zone code (eg, Z). So, the CF spec example on that web page is fully iso8601 compliant! But it never says so explicitly, and it seems to suggest that using a Z would not be correct. However, there's this statement: "The reference time string (appearing after the identifier since) may include date alone; date and time; or date, time, and time zone." So, a Z timezone code seems acceptable.

In addition, the SONAR-netCDF4 spec provides this time origin string "1601-01-01 00:00:00Z". 

SO: My suggestion, for now: "seconds since 1900-01-01 00:00:00Z"